app/poros/spells.rb

```diff
@@ -0,0 +1,185 @@
+class Spells
```

Comment:
This was a Bear, to say the least.  Nearly everyone was a litttttttle different in how they needed their things parsed, and reading from the tables took a while to figure out.